pandas/core/internals/managers.py

```diff
@@ -289,6 +289,7 @@ def get_dtypes(self) -> npt.NDArray[np.object_]:
         dtypes = np.array([blk.dtype for blk in self.blocks], dtype=object)
         return dtypes.take(self.blknos)
 
+    # TODO: This is being used in a few places
```

Comment:
I'm seeing arrays being used in a few places, if this should still be removed then I'm thinking it's best for a followup.

Reply:
I'm seeing this used in

 - DataFrame._is_homogeneous_type
 - DataFrame._reduce
 - DataFrame._reduce_axis1
 - DataFrame.shift
 - NDFrame._logical_func
 - NDFrame.astype

I'm thinking to just update the docstring for now, removing any mentions of the ArrayManager.